src/renderer/wg_engine/tvgWgRenderData.cpp

```diff
@@ -298,10 +298,19 @@ void WgRenderDataShape::updateMeshes(WgContext &context, const RenderShape &rsha
         } else if (cmd == PathCommand::Close) {
             polyline.close();
         } else if (cmd == PathCommand::CubicTo) {
+            WgPoint pt0 = polyline.pts.last().trans(rt->m);
```

Comment:
are you sure at this point that calling last() will not crash? you do this check later in appendCubic, but maybe it should be done here?